rosdep/python.yaml

```
@@ -6768,6 +6768,59 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian:
+    '*': [python3-google-auth]
+    stretch:
+      pip:
+        packages: [google-auth]
+  fedora: [python3-google-auth]
+  opensuse: [python3-google-auth]
+  rhel:
+    '*': [python3-google-auth]
+    '7': null
+  ubuntu: [python3-google-auth]
+python3-google-auth-httplib2:
+  debian:
+    '*': [python3-google-auth-httplib2]
+    buster:
+      pip:
+        packages: [google-auth-httplib2]
+    stretch:
+      pip:
+        packages: [google-auth-httplib2]
+  fedora:
+    pip:
+      packages: [google-auth-httplib2]
+  opensuse: [python3-google-auth-httplib2]
+  ubuntu:
+    '*': [python3-google-auth-httplib2]
+    bionic:
+      pip:
+        packages: [google-auth-httplib2]
+python3-google-auth-oauthlib:
+  debian:
+    '*': [python3-google-auth-oauthlib]
+    buster:
+      pip:
+        packages: [google-auth-oauthlib]
+    stretch:
+      pip:
+        packages: [google-auth-oauthlib]
+  fedora:
+    pip:
+      packages: [google-auth-oauthlib]
+  rhel:
+    '*': [python3-google-auth-oauthlib]
+    '7': null
+  ubuntu:
+    '*': [python3-google-auth-oauthlib]
+    bionic:
+      pip:
+        packages: [google-auth-oauthlib]
+    focal:
+      pip:
+        packages: [google-auth-oauthlib]
```

```suggestion
python3-google-auth-oauthlib:
  debian:
    '*': [python3-google-auth-oauthlib]
    buster: null
  rhel:
    '*': [python3-google-auth-oauthlib]
    '7': null
  ubuntu:
    '*': [python3-google-auth-oauthlib]
    bionic: null
    focal: null
```

With the same rationale as above I suggest removing the pip definitions entirely.